### ISSUE TITLE
Custom poll input options not announced individually

### ISSUE THREAD

**Issue by KDSBrowne** [bigbluebutton/bigbluebutton#6657]
When the custom poll button is selected, navigating to the first "add poll option" with the virtual cursor will cause the screen reader to announce all the contained items as one.
